### PR TITLE
feature/v2.8.3

### PR DESCRIPTION
**Pull request by mattgresham**
- Bump library version
- Bump vulnerable packages
- Add the ability to report Spice version (SP-677)
- Fix test not waiting for promise to complete before continuing

### CODE REVIEW

File: src/Spi.js [hunk 1]
```diff
@@ -337,6 +338,19 @@ class Spi {
         this._posVersion = posVersion;
     }
 
+    ResendPosInfo()
```
Comment (RaymondK-mx51) [inline src/Spi.js:341]: We have a `SetPosInfo` and a `ResendPosInfo` both are public functions and they both have same purpose but executed differently. 

As someone integrating with the function, which one would I use? When would I use ResendPosInfo?

**Review by RaymondK-mx51 — CHANGES_REQUESTED** [review]

File: src/Spi.js (hunk 1 above)
Comment (RaymondK-mx51) [reply src/Spi.js:341]: As discussed on Zoom

Can you please rename `ResendPosInfo` to `SetAdaptorPosInfo` and `SetSpiceInfo` to `SetAdaptorInfo`

Comment (mattgresham) [reply src/Spi.js]: Updated!

**Review by RaymondK-mx51 — APPROVED** [review]

**Comment by RaymondK-mx51** [comment]
Thanks Matt!

**Pull request merged**